src/simulation/dynamics/spinningBodies/spinningBodiesOneDOF/_UnitTest/test_spinningBodyOneDOFStateEffector.py

```diff
@@ -65,11 +63,15 @@ def test_spinningBodyTest(show_plots):
 
     against their initial values.
     """
-    [testResults, testMessage] = spinningBody(show_plots)
+    [testResults, testMessage] = test_spinningBody(show_plots, cmdTorque, lock)
     assert testResults < 1, testMessage
 
 
-def spinningBody(show_plots):
+@pytest.mark.parametrize("cmdTorque, lock", [
+    (0.0, False)
+    , (0.0, True)
+])
+def test_spinningBody(show_plots, cmdTorque, lock):
```

Comment:
move the parameterize definitions to above  `test_...` method

Reply:
Done.